ERCS/erc-7250.md

```diff
@@ -0,0 +1,160 @@
+---
+eip: 7250
+title: Nitro Protocol for State Channels with Account Abstraction
```

Comment:
ERCs are for Ethereum standardization and not product/protocol promotion. 

```suggestion
title: State Channels with Account Abstraction
```
Can this ERC be used by any state channel implementation, or is it limited to Nitro based state channels?

Reply:
Hello,
There is no commercial name or product nitro,
this can be used by most state-channels

We are from the open-source project https://statechannels.org/
https://github.com/statechannels

We are mentioned in https://ethereum.org/en/developers/docs/scaling/state-channels/

I will remove nitro, as it's an historical name of the open source protocol, but we also have the following name: "Stateful Asset Transfer Protocol"